Goal: Fix Python build for 3.0.25c

Status wrt upstream: Fixed in SVN

Index: samba-3.0.25c/source/python/py_smb.c
===================================================================
--- samba-3.0.25c.orig/source/python/py_smb.c
+++ samba-3.0.25c/source/python/py_smb.c
@@ -48,7 +48,7 @@
 
 	ZERO_STRUCT(ip);
 
-	if (!cli_connect(cli, server, &ip))
+	if (!NT_STATUS_IS_OK(cli_connect(cli, server, &ip)))
 		return NULL;
 
 	return new_cli_state_object(cli);
